src/dra/cd.c

```diff
@@ -781,13 +782,13 @@ void UpdateCd(void) {
             break;
 
         case CdCallback_16:
-            D_80137FB0 = (s16)func_80021350(
+            D_80137FB0 = (s16)SsVabOpenHeadSticky(
                 g_Cd.D_80137F88, D_80137F94, D_800BD1C8[D_80137F94]);
```

Comment:
// Set up a VAB header list in main memory
D_80137F88 -> vabHeaderStartAddress
D_80137F94 -> vabID
D_800BD1C8 -> soundBufferAddrs
D_80137F94 -> curSoundbufferAddr